camelConnector/src/test/java/io/vantiq/extsrc/camelconn/connector/VantiqConfigurationTest.java

```diff
@@ -70,9 +72,6 @@ void performConfigTest(String appName, String route, String routeFormat,
     void performConfigTest(String appName, String route, String routeFormat,
                            List<Map<String, Object>> compInitProps, Properties propertyValues, Verifier vfy,
                            List<String> rawReq) {
```

Comment:
Move this to specific tests.  This lets more tests run on Jenkins w/o salesforce crews.